HISTORY.md

```diff
@@ -1,3 +1,8 @@
+3.11.1 / 2017-11-02
```

Comment:
Could you change this to say "unreleased"? It may or may not be part of `3.11.1` and likely won't be released on November 2.